packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -90,39 +85,86 @@ module.exports = function (defaults) {
   });
 }
 
-async function embroiderInRepoAddon(project) {
-  await embroider(project);
-  await addInRepoAddon(project, 'fake-addon');
-}
+// async function embroiderInRepoAddon(project) {
+//   await embroider(project);
+//   await addInRepoAddon(project, 'fake-addon');
+// }
 
-async function addInRepoAddon(project, name, version = '0.0.0') {
+async function addInRepoAddon(project, names, version = '0.0.0') {
   project.linkDependency('ember-add-in-repo-tests', {
     baseDir: __dirname,
   });
 
   project.pkg['ember-addon'] = {
-    paths: [`lib/${name}`],
+    paths: [],
   };
 
-  merge(project.files, {
-    lib: {
-      [name]: {
-        'package.json': {
-          name,
-          version,
-          keywords: ['ember-addon'],
-        },
-        'index.js': `module.exports = {
-          name: require("./package").name,
-        };`,
-        tests: {
-          unit: getTestFiles('with-hooks-assert-includes-test.js'),
+  const lib = {};
+
+  names.forEach((name) => {
+    project.pkg['ember-addon'].paths.push(`lib/${name}`);
+    lib[name] = {
+      'package.json': `{
+        "name": "${name}",
+        "version": "${version}",
+        "keywords": ["ember-addon"]
+      }`,
+      'index.js': `module.exports = {
+        name: '${name}',
```

Comment:
Why is this change necessary?

Reply:
IMO `${name}` is easier to read and understand vs `require("./package").name`. Is there a reason behind using `require("./package").name`? Maybe I'm unfamiliar and missing something, why we should use 'require' here.

Reply:
It's the [default in the addon's blueprint](https://github.com/ember-cli/ember-addon-output/blob/master/index.js#L4). We're trying to adhere to it as closely as possible, which is the entire point. We're attempting to recreate as close as possible the _actual_ structure that we want to test. If you divert from that and make up your own structure, it's not testing as close to the real thing as possible.

Reply:
Ah! Thank you. Yes, we want to adhere to as-close-as-possible actual structure. Reverted to use require. Thanks.